Client/mods/deathmatch/logic/CClientColModel.h

```diff
@@ -22,17 +24,68 @@ class CClientColModel : public CClientEntity
 
     eClientEntityType GetType() const { return CCLIENTCOL; }
 
-    bool LoadCol(const SString& strFile, bool bIsRawData);
-    bool Load(bool isRaw, SString input);
-
-    bool IsLoaded() { return m_pColModel != NULL; };
+    bool           LoadCol(const SString& strFile, bool bIsRawData);
+    bool           Load(bool isRaw, SString input);
+    unsigned short GetVerticesCount() const { return m_usVerticesCount; };
+    void           SetCollisionHasChanged(bool bChanged) { b_hasChanged = bChanged; };
+    bool           HasChanged() const { return b_hasChanged; };
+    bool           IsLoaded() { return m_pColModel != NULL; };
 
     bool Replace(unsigned short usModel);
     void Restore(unsigned short usModel);
     void RestoreAll();
 
     bool        HasReplaced(unsigned short usModel);
     static bool IsCOLData(const SString& strData);
+    static bool CheckVector(CVector& vec, float fRadius = 0)
+    {
+        if (fRadius > 0)
+        {
+            return (MAX_COLLISION_SIZE >= vec.fX + fRadius && vec.fX + fRadius >= -MAX_COLLISION_SIZE && MAX_COLLISION_SIZE >= vec.fY + fRadius &&
+                    vec.fY + fRadius >= -MAX_COLLISION_SIZE && MAX_COLLISION_SIZE >= vec.fZ + fRadius && vec.fZ + fRadius >= -MAX_COLLISION_SIZE) &&
+                   (MAX_COLLISION_SIZE >= vec.fX - fRadius && vec.fX - fRadius >= -MAX_COLLISION_SIZE && MAX_COLLISION_SIZE >= vec.fY - fRadius &&
+                    vec.fY - fRadius >= -MAX_COLLISION_SIZE && MAX_COLLISION_SIZE >= vec.fZ - fRadius && vec.fZ - fRadius >= -MAX_COLLISION_SIZE);
+        }
+        else
+        {
+            return (MAX_COLLISION_SIZE >= vec.fX && vec.fX >= -MAX_COLLISION_SIZE && MAX_COLLISION_SIZE >= vec.fY && vec.fY >= -MAX_COLLISION_SIZE &&
+                    MAX_COLLISION_SIZE >= vec.fZ && vec.fZ >= -MAX_COLLISION_SIZE);
+        }
```

Comment:
else should not follow this return here. 
```suggestion
        return (MAX_COLLISION_SIZE >= vec.fX && vec.fX >= -MAX_COLLISION_SIZE && MAX_COLLISION_SIZE >= vec.fY && vec.fY >= -MAX_COLLISION_SIZE &&
                    MAX_COLLISION_SIZE >= vec.fZ && vec.fZ >= -MAX_COLLISION_SIZE);
```